lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,9 +8,13 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
```

Comment:
To add `attr_accessor :#{column}_data_uri`

Able to use `image_data_url` at form just to add `mount_uploader :image, ImageUploader`.